src/controllers/index.js

```diff
@@ -5,6 +5,8 @@ const signup = require("./signup");
 const login = require("./login");
 router.get("/", home.get);
 router.get("/login", login.get);
+router.post("/login", login.post);
 router.get("/signup", signup.get);
+router.get("/user/:name", home.get);
```

Comment:
Do we still need a homepage with '/' endpoint ,in line 6, when we actually give every user his/her own endpoint that gives him/her authorities to add messages after logging in?! :thinking: :thought_balloon: